tradefiles/sgx_parser.py

```
@@ -32,77 +30,101 @@ def parse_io_xlsx(input_path, output_path):
+                    bs = 'B' if quantity_float > 0 else 'S'
+                except (ValueError, TypeError):
+                    continue
+
+                spread = 'S' if "spread" in str(counterparty).lower() else ''
+                counterparty_first_word = str(counterparty).split(" ")[0]
+
+                tradetime = ""
+                if hasattr(time_val, 'hour'):
+                    tradetime = time_val.strftime('%H:%M:%S')
+                elif isinstance(time_val, datetime.datetime):
+                    tradetime = time_val.strftime('%H:%M:%S')
```

Ruff (0.12.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)